src/main/java/frc/robot/subsystems/swerve/SwerveModuleIoTalonFx.java

```diff
@@ -158,12 +157,13 @@ public void setTurnVoltage(double volts) {
 
   @Override
   public void updateInputs(SwerveModuleIoInputs inputs) {
+    
     BaseStatusSignal.refreshAll(
         driveMotorPosition,
         driveMotorVelocity,
         driveMotorAppliedVolts,
         driveMotorCurrent,
-        cancoderAbsolutePosition,
+        cancoderOffsetPosition,
```

Comment:
This instance shouldn't be renamed, since the true reading (status signal) is the absolute position.

Reply:
Agreed. Done. :)